packages/plugin-node/src/index.ts

```diff
@@ -481,10 +493,36 @@ export async function buildEntrypoint({
     const finishPromise = once(ws, 'finish');
     file.toStream().pipe(ws);
     await finishPromise;
+
+    // The `handler` will be `.output/server/pages/api/subdirectory/___vc/__launcher.launcher`
+    // or `.output/server/pages/api/___vc/__launcher.launcher`.
+    // This means everything has to be mounted to the `dirname` of the entrypoint.
+    nftFiles.push({
+      input: relative(dirname(nftOutput), outPath),
+      output: join('.output', 'server', 'pages', dirname(entrypoint), filename),
+    });
+  }
+
+  await fsp.writeFile(
+    nftOutput,
+    JSON.stringify({
+      version: 1,
+      files: nftFiles,
+    })
+  );
+
+  await fsp.copyFile(
+    join(outputWorkPath, renameTStoJS(entrypoint)),
+    pageOutput
+  );
+
+  let pageKey = relative(pagesDir, pageOutput);
+  if (process.platform === 'win32') {
+    pageKey = pageKey.replace(/\\/gm, '/');
```

Comment:
In the future, you can use this helper function:

https://github.com/vercel/vercel/blob/82924bb5c4e51bbb387dd0f6207d2bff6af90eca/packages/build-utils/src/fs/normalize-path.ts#L6-L8